static/js/stat_var_hierarchy/node_header.tsx

```diff
@@ -80,5 +88,15 @@ export class StatVarHierarchyNodeHeader extends React.Component<StatVarHierarchy
       </div>
     );
   }
+
+  private mouseMoveAction = (e) => {
```

Comment:
Is there a way to copy the text from the tooltip? We have been wanting to copy the dcid of the svg from hierarchy.

Reply:
that will require some thought about the design because the tooltip stays to the right of the mouse as the mouse moves, but to copy we would have to highlight the tooltip ..  maybe we can open a github issue for this?